docs/integrations/sources/twilio.md

```diff
@@ -73,6 +73,7 @@ The Twilio source connector supports the following [sync modes](https://docs.air
 * [Transcriptions](https://www.twilio.com/docs/voice/api/recording-transcription?code-sample=code-read-list-all-transcriptions&code-language=curl&code-sdk-version=json#read-multiple-transcription-resources)
 * [Usage Records](https://www.twilio.com/docs/usage/api/usage-record#read-multiple-usagerecord-resources) \(Incremental\)
 * [Usage Triggers](https://www.twilio.com/docs/usage/api/usage-trigger#read-multiple-usagetrigger-resources)
+* [Conversations](https://www.twilio.com/docs/conversations/api/conversation-resource#read-multiple-conversation-resources) \(Incremental\)
```

Comment:
I don't think it is incremental right now. If it should be, we need to inherit from another class.